.github/workflows/server.yaml

```
@@ -18,15 +18,31 @@ jobs:
+        # Health check to ensure postgres is ready before starting the tests.
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
     permissions:
       pull-requests: write
     defaults:
       run:
-        working-directory: ./server
+        working-directory: ./serverenv:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 40: mapping values are not allowed in this context

(syntax-check)